components/card.js

```diff
@@ -27,4 +27,21 @@ export const CardContent = styled.div`
   padding: 40px;
 `;
 
+// padding: top, right, bottom, left
+// jsutify contente space around
+export const CardDiv = styled.div`
```

Comment:
let's keep this consistent with the rest
```suggestion
export const CardContainer = styled.div`
```